### ISSUE TITLE
Tabs: cannot focus any tab when no value/defaultValue is set

### ISSUE THREAD

**Issue by var-poro** [chakra-ui/zag#2762]
# 🐛 Bug report

When Tabs.Root is rendered without value or defaultValue, none of the Tabs.Trigger elements are focusable. As a result, keyboard users cannot focus or navigate the tabs until a value is programmatically selected, which is an accessibility issue.

## 💥 Steps to reproduce

1. Create a default example Tabs.
2. Try to reach a tab with the keyboard (Tab or arrow keys).
3. No tab trigger receives focus; keyboard users can’t interact with the tabs.

## 💻 Link to reproduction

Ark's default example: https://stackblitz.com/edit/kx3f6emy?file=src%2FApp.tsx

## 🧐 Expected behavior

When no tab is selected, the first non-disabled Tabs.Trigger should be focusable (tabIndex=0) so keyboard users can reach the tablist and then use arrow keys to navigate.

## 🌍 System information

| Software         | Version(s) |
| ---------------- | ---------- |
| Zag Version      |     1.24.2       |
| Browser          |       Chromium 141.0.7390.55     |
| Operating System |      Mac OS 15.7    |

Thanks!

**Comment by segunadebayo** [comment]
Even if we fix this, it will still be incorrect in SSR before hydration, as we need to query the DOM for this.

I recommend that you ensure the tabs component in your design system sets a default value.
